README.md

```diff
@@ -16,14 +16,26 @@ Navigate to the sample on your machine at the command prompt or terminal. Make s
 **Build and run Docker image**
 
  - IIS Hosted WCF service
+   - Open the 4.6.2/WcfBasicIISHost/WcfServiceTest.sln in Visual Studio and build the Release project locally.
    - Open a powershell window
    - Go to the sample directory that contains the dockerfile for that sample, such as 4.6.2/WcfBasicIISHost
-   - Build the Docker image: "docker build -t vmwcfbasichttpiishost ."
-   - Run the IIS hosted WCF service in the container: "docker run -d -p 83:83 --name wcfbasichttpiishost vmwcfbasichttpiishost"
+   - Build the Docker image: "docker build -t wcfbasichttpiishost ."
+   - Run the IIS hosted WCF service in the container: "docker run -d -p 83:83 --name wcfbasichttpiishost wcfbasichttpiishost"
  - Selfhosted WCF service
    - Open a powershell window
    - Go to the sample directory that contains the dockerfile for that sample, such as 4.6.2/WcfBasicHttpSelfHost
-   - Build the Docker image: "docker build -t vmwcfbasichttpselfhost ."
-   - Run the selfhosted WCF service in the container: "docker run -a stdin -a stdout -i -p 83:83 --name wcfbasichttpselfhost vmwcfbasichttpselfhost"

```

Comment:
@Lxiamail Should I indent the TCP instructions under a "Self Hosting WCF" section then? I can do that.